api/base_controller.py

```diff
@@ -0,0 +1,81 @@
+import flask
```

Comment:
All this code is refactored from `CirculationManagerController` in `controller.py` and moved to a new file to avoid a circular dependency. I wanted to keep all the Adobe-related controllers in `adobe_vendor_id.py`, so I needed a new place to put code that authenticates patrons according to the circulation manager's configuration (as opposed to the technique defined by Adobe's vendor ID spec).